app/models/get_next.rb

```diff
@@ -0,0 +1,11 @@
+class GetNext
+  def initialize
```

Comment:
This is nicer makes sense to be its own thing - and will be easier to implement the real Algo 👍